project_name/tests/test_federica.py

```diff
@@ -0,0 +1,8 @@
+def test_federica(test):
```

Comment:
The parameter `test` in your function here will is something that `pytest` will complain about.
It's because "test" isn't defined elsewhere.

Take a look at the example PR that shows how to parametrize the input - it's rather neat:
https://github.com/pcdshub/pcds-ci-test-repo-python/pull/6/files#diff-822bb6cd496579f843d5d333f49b1a4b0b78b43311dfb19d73fa37a7d0a2da14R10-R17

It says "use these values for the input parameter value and expect this to be the result". You write the test once and use a bunch of input parameters.